database/api/routes.md

```diff
@@ -0,0 +1,122 @@
+# api/v1/save/
+
+### Request Types(s)
++ POST
+
+### Request Body
+NOTE:: recon json key has been removed from previous versions
+```
+{
+    username: String
+    title: String
+    description: String
+    model: String
+    modelChecksum: String
+    graph: String
+    tags: [ String ]
+    scripts: [
+        { 
+            name: String
+            content: String
+            checksum: String
+        }
+    ]
+    files: [
+        {
+            name: String
+            uri: String
+            size: Integer
+            checksum: String
+            lastModified: String
+        }
+    ]
+    programBlock: [
+        {
+            id: Integer
```

Comment:
is this id the id or the version id?

Reply:
id. it is the id of the `data` table. the primary key of the `data` table on the server will be a composite key of the key from the database on yesworkflow, and its assciated version id